app/controllers/application_controller.rb

```diff
@@ -66,9 +66,11 @@ def hide_intercom?
       ["StateFile::StateFilePagesController", "privacy_policy"],
       ["StateFile::FaqController", ""]
     ]
-    state_file? && pages_to_hide_from.any? do |controller, action|
-      self.class.name == controller && (action == "" || action_name == action)
-    end
+    is_hidden_page = state_file? && pages_to_hide_from.any? do |controller, action|
+                      self.class.name == controller && (action == "" || action_name == action)
+                    end
```

Comment:
unrelated, but what's the logic behind hiding intercom on those pages?

Reply:
@arinchoi03 like why do we hide it from those pages? I'm not sure but maybe I can dig into some pivotal story history for context...

Reply:
np, it's not a big deal 🤷 just wondering aloud...